src/rngs/mod.rs

```diff
@@ -0,0 +1,184 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// https://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Various RNGs and wrappers.
+//!
+//! This module provides the three standard RNGs exposed by Rand (besides
+//! specific PRNG algorithms in the `prng` module). Those are [`ThreadRng`],
+//! [`StdRng`], and [`SmallRng`].
```

Comment:
`ThreadRng` is not really a *standard* generator but merely a handle... I think it would be better to say *the two standard PRNGs*